dropwizard-index-page/src/test/java/com/palantir/indexpage/ClassPathAssetTests.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2016 Palantir Technologies, Inc. All rights reserved.
+ */
+
+package com.palantir.indexpage;
+
+import static org.junit.Assert.assertNotNull;
+
+import com.google.common.io.Resources;
+import org.junit.Test;
+
+/**
+ * Tests for {@link ClasspathAsset}.
+ */
+public final class ClassPathAssetTests {
```

Comment:
these tests only check for arguments. what about all the logic inside of this class?